inspectors/types/articulationtypes.h

```diff
@@ -7,11 +7,22 @@ class ArticulationTypes
 {
     Q_GADGET
 
-    Q_ENUMS(Style)
+    Q_ENUMS(Placement)
+    Q_ENUMS(Direction)
+
 public:
-    enum Style {
-        STYLE_STANDART = 0,
-        STYLE_BAROQUE
+    enum Placement {
+        TYPE_ABOVE_STAFF,
+        TYPE_BELOW_STAFF,
+        TYPE_CHORD_AUTO,
+        TYPE_ABOVE_CHORD,
+        TYPE_BELOW_CHORD
+    };
+
+    enum Direction {
+        AUTO,
+        DOWN,
+        UP
```

Comment:
1. Please use `enum class`;
2. Please use `Q_ENUM` rather than `Q_ENUMS`, as the latter is deprecated. Also, put `Q_ENUM` behind the enum type declaration.
Same for the other `types` file. See https://github.com/vpereverzev/MuseScore/pull/10.